extensions/ql-vscode/src/query-testing/test-ui.ts

```diff
@@ -29,7 +15,7 @@ class QLTestListener extends DisposableObject {
     this.push(adapter.testStates(this.onTestStatesEvent, this));
   }
 
-  private onTestStatesEvent(_e: VSCodeTestEvent): void {
+  private onTestStatesEvent(): void {
```

Comment:
This is an odd case because this is essentially unused code, in that we're deliberately implementing this to do nothing. I think we can safely remove the argument here without losing any information or risking the plumbing breaking down, because we're the interface defined by the `testStates` function, and the `TestRunStartedEvent` type and friends are coming from `vscode-test-adapter-api` dependency.

Essentially we're not actually deleting any code here and it would be possible to recreate this by looking at the types from the `vscode-test-adapter-api` dependency.